frontend/src/components/Dropdown/Dropdown.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyledDropdown, DropdownItem, Image} from "./Dropdown.styled";
+import { FaBell, FaFlag, FaEyeSlash } from "react-icons/fa";
+
+const Dropdown = () => {
+  const dropdownItems = [
+    {
+      icon: <FaBell />,
+      text: 'Notify me'
+    },
+    {
+      icon: <FaFlag />,
+      text: 'Report this post'
+    },
+    {
+      icon: <FaEyeSlash />,
+      text: "Don't show me this post"
+    }
+  ]
+  const displayDropdownItems = dropdownItems.map((dropdownItem) =>
+    <DropdownItem key={dropdownItem.text}>
+      <Image>
+        {dropdownItem.icon}
+      </Image>
+        {dropdownItem.text}
+    </DropdownItem>
+  )
+
+
+  return (
+    <StyledDropdown>
+      {displayDropdownItems}
+    </StyledDropdown>
+    )
+}
+
+export default Dropdown;
```

Comment:
Tak samo brak znaku końca nowej linii w każdym pliku, przez co Github pokazuje tę czerwoną ikonkę - prettier powinien to pokazywać